apis/Google.Cloud.ErrorReporting.V1Beta1/Google.Cloud.ErrorReporting.V1Beta1/ReportErrorsServiceClient.g.cs

```diff
@@ -253,7 +253,7 @@ public static ReportErrorsServiceClient Create(grpccore::CallInvoker callInvoker
         public virtual ReportErrorsService.ReportErrorsServiceClient GrpcClient => throw new sys::NotImplementedException();
 
         /// <summary>
-        /// Report an individual error event.
+        /// Report an individual error @event.
```

Comment:
Don't want `@` here, but it's probably OK for now. I'll fix the generator soon, then this can be re-gened.

Reply:
Yup, that's definitely simpler than trying to fix that. Well spotted though :)